x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -80,25 +79,64 @@ describe.skip('Endpoints page', { tags: ['@ess', '@serverless'] }, () => {
       .should('have.text', createdHost.hostname);
   });
 
+  it('should update endpoint policy on Endpoint', () => {
+    const parseRevNumber = (revString: string) => Number(revString.match(/\d+/)?.[0]);
+
+    loadPage(APP_ENDPOINTS_PATH);
+
+    cy.getByTestSubj('policyListRevNo')
+      .first()
+      .invoke('text')
+      .then(parseRevNumber)
+      .then((initialRevisionNumber) => {
+        // Update policy
+        cy.getByTestSubj('policyNameCellLink').first().should('exist').click();
+
+        cy.getByTestSubj('endpointPolicyForm-malware-blocklist-enableDisableSwitch').as(
+          'blocklistSwitch'
+        );
+        cy.get('@blocklistSwitch').should('exist').click();
+        cy.get('@blocklistSwitch').should('be.enabled');
+
+        cy.getByTestSubj('policyDetailsSaveButton').should('be.visible').click();
+        cy.getByTestSubj('policyDetailsConfirmModal').should('be.visible').click();
+        cy.getByTestSubj('confirmModalConfirmButton').should('be.visible').click();
+        cy.contains(/has been updated/);
+
+        cy.getByTestSubj('policyDetailsBackLink').should('be.visible').click();
+
+        // Assert disappearing 'Out-of-date' indicator, Success Policy Status and increased revision number
+        cy.getByTestSubj('rowPolicyOutOfDate', { timeout: 10000 }).should('not.exist'); // depends on the 10s auto-refresh
```

Comment:
Look into cypress-recurse for more control over timeouts on dom elements. Example https://github.com/elastic/kibana/blob/9994d0535317382e19886cfc5cc7d9879fd9642f/x-pack/plugins/osquery/cypress/tasks/response_actions.ts#L53-L61

Reply:
Thanks for the heads up. Since I don't need finer control here I'm going to leave it as-is but will definitely store this info for any future needs.